server/models/Action.js

```diff
@@ -19,6 +19,11 @@ var ActionSchema = new Schema({
     type: String,
     enum: ['rated','watched','followed'],
     required: true
+  },
+  hide: {
+    type: Boolean,
+    default: false,
+    required: true
   }
```

Comment:
Falta adicionar uma referencia pra a ação em si (watched/rating/followed)
_action: {
type: String,
required: true
}

Reply:
I think the field _action_ already has this purpose, no?

Reply:
não

Reply:
name é um Date mesmo?

Reply:
I don't know, @Rafaela9136. I just changed from `date` to `type` because I thought the person who implemented this didn't pay attention. There was not type declared for `name`.